velox/exec/tests/utils/HiveConnectorTestBase.cpp

```diff
@@ -107,20 +107,25 @@ std::vector<std::shared_ptr<connector::hive::HiveConnectorSplit>>
 HiveConnectorTestBase::makeHiveConnectorSplits(
     const std::string& filePath,
     uint32_t splitCount,
-    dwio::common::FileFormat format) {
+    dwio::common::FileFormat format,
+    int64_t maxCoalesceBytes,
```

Comment:
why is maxCoalesceBytes int64 and maxCoalesceDistance int32? Maybe have them both as int32?

Reply:
The existing code used them this way. I choose not to open the can of worms to change the types. :)